src/msgs/staking/query/mod.rs

```diff
@@ -12,7 +14,9 @@ pub enum QueryMsg {
     #[cfg_attr(feature = "cosmwasm", returns(StakerAndSeq))]
     GetStakerAndSeq { public_key: String },
     #[cfg_attr(feature = "cosmwasm", returns(bool))]
-    IsExecutorEligible { proof: String, dr_id: String },
+    IsExecutorCommitteeEligible { public_key: String },
+    #[cfg_attr(feature = "cosmwasm", returns(bool))]
+    IsExecutorEligible(is_executor_eligible::Query),
```

Comment:
To follow the rest of queries, why not like this?

```
IsExecutorEligible{ data: is_executor_eligible::Query},
```

Reply:
It could be done that way, but our standard practice for messages is that if something is complicated to move to its type, we've been using tuples for it.

If we did this, it would change the `json` as well from:
```json
"is_executor_eligible": {
  "data": data,
}
```

to
```json
"is_executor_eligible": {
  "data": {
    "data": data,
  }
}
```